src/GafferScene/SceneAlgo.cpp

```diff
@@ -489,7 +505,6 @@ SceneAlgo::History::Ptr historyWalk( const CapturedProcess *process, InternedStr
 		if( scene && plug == scene->getChild( scenePlugChildName ) )
 		{
 			ContextPtr cleanContext = new Context( *process->context );
-			cleanContext->remove( SceneAlgo::historyIDContextName() );
```

Comment:
It looks like `cleanContext` is unecessary now, and we can just reference `process->context` directly.